tests/modules/programs/waybar/package-null.nix

```diff
@@ -0,0 +1,19 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+{
+  config = {
+    home.stateVersion = "23.11";
+
+    programs.waybar = {
+      package = null;
+      enable = true;
+    };
+
+    nmt.script = ''
+      assertPathNotExists home-files/.config/waybar/style.css
```

Comment:
hum shouldn't it check that the executable "wayland" is not available instead ?

Reply:
@teto I agree, but I need some help :) I was looking at other tests and did not find if there is already a test does this for other programs. This test would anyway fail on the old code, as `null` was not accepted at all.

Any hints please how to test that waybar package would not be installed?

Reply:
@rycee hi, maybe some clues from you, I am a bit new to home-manager and nix :(

Reply:
just checking `which waybar` return code should be ok ?

Reply:
@teto I also had to define the package stub to actually do something :)

It should be resolved, please check!